lib/ansible/playbook/playbook_include.py

```diff
@@ -144,8 +147,9 @@ def _preprocess_import(self, ds, new_ds, k, v):
         if len(items) == 0:
             raise AnsibleParserError("import_playbook statements must specify the file name to import", obj=ds)
         else:
-            new_ds['import_playbook'] = items[0]
+            new_ds['import_playbook'] = items[0].strip()
             if len(items) > 1:
+                display.warning('Additional parameters in import_playbook statements are no longer supported and will be removed in version 2.14')
```

Comment:
```suggestion
                display.warning('Additional parameters in import_playbook statements are not supported. This will be an error in version 2.14')
```